### ISSUE TITLE
Example for authentification

### ISSUE THREAD

**Issue by spiderbit** [iv-org/documentation#127]
Let's say I have a account and want the Feed or my subscriptions, what would I have to do, using curl?

There is 1 default Token that comes with the installation do I use that to directly get the feed or do you I create a new token with that token as authentication?

I just understand or can make it work, I am no oauth expert I assume it's based ot that right?

**Comment by unixfox** [comment]
Probably a duplicate of https://github.com/iv-org/invidious/issues/473

**Comment by spiderbit** [comment]
@unixfox I don't think so, I just want a example and if you want to see it as bug for the documentation adding a curl example to the docu would probably not hurt.

The linked bug don't ask for code example or something alike, it was more the request to have a api, I want a easy quickstart guide for it basically. Sure you could maybe learn from the bug how things are supposed to work which post / get commands I should send, but not exactly step by step with all parameters.

I don't even know when I want let's say my subscription from 0, what I have to do in which order, and do I need to use the password for registering a token or can I use the standard token to create a new token, or whatever... it's very confusing to me.

**Comment by unixfox** [comment]
https://github.com/iv-org/invidious/issues/473 is about implementing a way to fetch subscriptions using a token for example, exactly what you are asking.

If this feature doesn't exist yet, we can't provide you a documentation in order to use it.

**Comment by syeopite** [comment]
We do have documentation for authenticated endpoints! https://github.com/iv-org/documentation/blob/master/Authenticated-Endpoints.md#get-apiv1authsubscriptions

**Comment by spiderbit** [comment]
Yes but a api if you don't understand even which steps in which order you have to take is not enough. Example code is often very helpful. I am sure that if you have written 5-10 clients for OAuth (which it does not even officially refer to, but seems to be what it is?) this API doku is sure enough to implement it. But if you never done it, then this API dokumentation is very confusing.

If you think I am the only developer on the world that is to stupid to get it not working with that doku write the few curl lines needed and close the bug.

If it's so easy it shouldn't be that much work to paste such lines, I am flexible take also code in other languages like Python or something. Or point me to any opensource client that implemented that auth features?
